quipucords/api/migrations/0001_initial.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.11.5 on 2018-04-19 15:22
+# Generated by Django 1.11.5 on 2018-08-02 20:05
```

Comment:
No changing migration 0001 :).  All existing migrations should remain unchanged.  You will add a new migration.